src/unity/python/turicreate/visualization/plot.py

```diff
@@ -0,0 +1,29 @@
+import time
+import json
+
+class Plot(object):
+    def __init__(self):
+        self.plot_object = tc.extensions._Plot()
```

Comment:
Where does `tc` come from here? Doesn't it need to be imported in this file?

Reply:
I changed the way we access Plot. It was getting tricky to integrate with the existing cython bindings we had for SFrame and SArray. If you think there's a better way of doing it. I'd be more than happy to change it!